website/server/controllers/api-v3/members.js

```diff
@@ -82,9 +83,34 @@ api.getMember = {
  * @apiSuccess {Number} data.*.achievements.index The unique index assigned to the achievement (only for sorting purposes)
  * @apiSuccess {Anything} data.*.achievements.value The value related to the achievement (if applicable)
  * @apiSuccess {Number} data.*.achievements.optionalCount The count related to the achievement (if applicable)
+
+ * @apiSuccessExample {json} Successful Response
```

Comment:
I think the example should have the full depth of the actual response that would be returned, but not the full breadth.  Just a few entries would be enough.  Might not even need to do all the categories.  But I think omitting levels in the response object is potentially very misleading.